doc/building.md

```diff
@@ -1804,9 +1805,17 @@ temporarily.
 On Windows, when configuring, `fixpath.sh` may report that some directory names
 have spaces. Usually, it assumes those directories have [short
 paths](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/fsutil-8dot3name).
-You can run `fsutil file setshortname` in `cmd` on certain directories, such as
-`Microsoft Visual Studio` or `Windows Kits`, to assign arbitrary short paths so
-`configure` can access them.
+You can run `fsutil file setshortname` in `cmd` on directories to assign
+arbitrary short paths so `configure` can access them. If the result says "Access
+denied", it may be that there are processes running in that directory; in this
+case, you can reboot Windows in safe mode and run the command on those directories
+again.
+
+The only directories required to have short paths are `Microsoft Visual Studio`
+and `Windows Kits`; the rest of the "contains space" warnings from `configure`
+can be ignored. You can choose any short name; once it is set, `configure`'s
```

Comment:
What other directories are reported from configure? 

It does not seem correct to me to say that they can just be ignored -- either the directory is relevant for compilation, and then we need to have it without spaces, or it is not really used and then it should be removed, or made optional.

If  `Microsoft Visual Studio` and  `Windows Kits` are typical path-with-spaces that users can run into, we should perhaps add an additional check to configure for these. In fact, I think what we *should* do is to have configure print this help about `fsutil file setshortname` if it encounters a path with spaces that it cannot turn into a short name. That way, a complete copy/pastable command line can be generated for the user to run.

Reply:
configure scans `PATH` which may include other spaced directories, such as "Intellij Idea." On a user device, this list can quickly become huge and it would be unrealistic for users to generate short paths for every single one of those directories.

In fact, the only 2 directories that need short path configuration for configure to work correctly are Microsoft Visual Studio and Windows Kits.

Reply:
Hm, it feels buggy that it should complain about random entries in the PATH; it should only be an issue if we try to FIXUP a path. I need to look into that.

Reply:
Can I leave this part as-is?

Reply:
Yes, leave it in this patch to the documentation. If I address the core issue in configure, I'll update the docs as well.

Reply:
For reference, I filed this: https://bugs.openjdk.org/browse/JDK-8338059